app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,32 @@
+
+
+
+
+
+
+<div class="footer-top">
+  <div class="footer-links-text" >
+    <a href="#"><i > Invest </i></a>
+    <a href="#"><i > Develop </i></a>
+    <a href="#"><i > About Us </i></a>
+    <a href="#"><i > Contact Us </i></a>
+  </div>
+  <div class="footer-contact" >
+    <div class = "footer-contact-text">
+    <p> T: +44 207 608 377 </p>
+    <p> E: homies@vestate.fund </p>
+    </div>
+  </div>
+</div>
+<div class="footer-bottom">
+  <div class="footer-links">
+    <a href="#"><i class="fab fa-instagram"></i></a>
+    <a href="#"><i class="fab fa-facebook"></i></a>
+    <a href="#"><i class="fab fa-twitter"></i></a>
+    <a href="#"><i class="fab fa-linkedin"></i></a>
```

Comment:
we're not actually setting up any social media pages for vestate, so do we need these links?